apps/L1E/instagram/instagram-frontend/src/app/(main)/(auth)/signin/page.tsx

```
@@ -0,0 +1,118 @@
+                                            )}
+                                        />
+                                    )
+                                })
+                            }
+                            <p className="text-sm font-medium text-[#2563EB] text-center py-[10px] px-[16px]">Forgot password?</p>
+                            <Button type="submit" className="w-full bg-blue-600/50">Log in</Button>
+                        </form>
+                    </Form>
+                </div>
+                <div className="bg-white rounded-[10px] py-4">
+                    <p className="text-center py-[10px] text-sm font-normal">Don't have an account? <a href="/signup" className="text-blue-600 font-bold pl-4">Sign up</a></p>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)